specs/sharding/beacon-chain.md

```diff
@@ -103,460 +101,377 @@ The following values are (non-configurable) constants used throughout the specif
 | - | - | - |
 | `PRIMITIVE_ROOT_OF_UNITY` | `7` | Primitive root of unity of the BLS12_381 (inner) modulus |
 | `DATA_AVAILABILITY_INVERSE_CODING_RATE` | `2**1` (= 2) | Factor by which samples are extended for data availability encoding |
-| `POINTS_PER_SAMPLE` | `uint64(2**3)` (= 8) | 31 * 8 = 248 bytes |
+| `POINTS_PER_SAMPLE` | `uint64(2**4)` (= 16) | 31 * 16 = 496 bytes |
```

Comment:
Does it make sense to rename to `FIELD_ELEMENTS_PER_SAMPLE` (following to `BLSPoint` => `BLSFieldElement` renaming)